exercises/practice/pangram/.approaches/bitfield/snippet.txt

```diff
@@ -0,0 +1,8 @@
+    for letter in sentence.chars() {
+        if letter >= 'a' && letter <= 'z' {
+            letter_flags |= 1 << (letter as u8 - A_LCASE);
+        } else if letter >= 'A' && letter <= 'Z' {
+            letter_flags |= 1 << (letter as u8 - A_UCASE);
+        }
+    }
+    letter_flags == ALL_26_BITS_SET
```

Comment:
```suggestion
for letter in sentence.chars() {
    if letter >= 'a' && letter <= 'z' {
        letter_flags |= 1 << (letter as u8 - A_LCASE);
    } else if letter >= 'A' && letter <= 'Z' {
        letter_flags |= 1 << (letter as u8 - A_UCASE);
    }
}
letter_flags == ALL_26_BITS_SET
```

Reply:
It seems no matter how conscientious I try to be about the indents they still get past me sometimes.

Reply:
No worries, now that GitHub suggestions support me using shift+tab, it is very easy to suggest the right indentation.